core/vm/analysis.go

```diff
@@ -76,7 +76,7 @@ func codeBitmapInternal(code, bits bitvec) bitvec {
 	for pc := uint64(0); pc < uint64(len(code)); {
 		op := OpCode(code[pc])
 		pc++
-		if op < PUSH1 || op > PUSH32 {
+		if int8(op) < int8(PUSH1) { // If not PUSH (the int8(op) > int(PUSH32) is always false).
```

Comment:
`If not PUSH (the int8(op) > int(PUSH32) is always false)`
why?

Reply:
The opcode of ["PUSH1" is "0x60"](https://ethervm.io/#60), and all the other `PUSH` opcodes have a greater opCode than "PUSH1".  An opcode that is less than "0x60" can never be a "PUSH32" .

Reply:
but `op > PUSH32` could be true right, say if op is Dup1(0x80)